src/routes/RemediationDetails.js

```diff
@@ -19,6 +25,7 @@ import ActivityTabUpsell from '../components/EmptyStates/ActivityTabUpsell';
 import DeniedState from '../components/DeniedState';
 import SkeletonTable from '../skeletons/SkeletonTable';
 import '../components/Status.scss';
+import instance from '@redhat-cloud-services/frontend-components-utilities/interceptors';
```

Comment:
Please use the axios instance via the `useAxiosWithInterceptors` hook.

Reply:
Reason for this being that the hook will use the chrome instance coming from the `useChrome` hook. 

This `instance` here will try to access chrome stuff via the `window.insights` object, which we should not do.